libs/pavex_cli/tests/ui_tests/reflection/crate_resolution/transitive_dependencies_can_be_renamed/expectations/diagnostics.dot

```diff
@@ -1,7 +1,7 @@
 digraph "GET /handler - 0" {
-    0 [ label = "dep::handler(http_0_1_21::header::HeaderName, http_0_2_12::HeaderName) -> pavex::response::Response"]
+    0 [ label = "dep::handler(http_0_1_21::header::HeaderName, http_0_2_11::HeaderName) -> pavex::response::Response"]
```

Comment:
Again, why has this rolled back?

Reply:
Possible that something might have lingered in the `ui_test_envs` folder. If you delete it and re-run `cargo test` it might work itself out.